UI/ChatGPT/src/ChatGPT/Services/ChatService.cs

```diff
@@ -95,4 +95,4 @@ private ChatCompletionCreateRequest ToCompletionRequest(ChatRequest request)
 			Model = Models.Gpt_3_5_Turbo
```

Comment:
We may want to upgrade to a newer, better, faster model ;-)

Reply:
@carldebilly not in this PR, we are keeping things the same as before to migrate to single project. We can make improvements in following PRs